recipe/meta.yaml

```diff
@@ -8,27 +8,24 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/robyn-{{ version }}.tar.gz
-  sha256: 759a5d94546c1bf3c858cb7e855f67ea009098953ec400f5a01a61fb47dcea60
+  sha256: 89e89799de78635e8b868d4fa396a66256f5920cd63191cf98362a2384f13188
 #   patches:
 #     - 0001-add-maturin.patch
 
 build:
-  skip: true  # [py<38]
+  skip: true  # [py<37]
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - {{ compiler('rust') }}
```

Comment:
You need to specify the rust compiler anyway, regardless if maturin is added as a build dependency.